infrastructure/cicsk8s/galasa-build/java-archive/values.yaml

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright contributors to the Galasa project
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+# These values are overridden in the ArgoCD app.
```

Comment:
Would it be worth mentioning which ArgoCD app and where to find it?

Reply:
Info about the app and where to find it added